locales/fa/validation-inline.php

```diff
@@ -36,8 +36,8 @@
     'date'                 => 'این مقدار یک تاریخ معبتر نیست.',
     'date_equals'          => 'این مقدار باید یک تاریخ مساوی با :date باشد.',
     'date_format'          => 'این مقدار با فرمت :format همخوانی ندارد.',
-    'declined'             => 'This value must be declined.',
-    'declined_if'          => 'This value must be declined when :other is :value.',
+    'declined'             => 'این مقدار قابل پذیرش نیست',
```

Comment:
We missed the period at the end of the sentence.

Reply:
I will fix it